prusa/link/web/cameras.py

```diff
@@ -26,20 +29,46 @@ def photo_by_camera_id(camera_id):
         return JSONResponse(status_code=state.HTTP_NO_CONTENT,
                             message=f"Camera with id: {camera_id} did not "
                                     f"take a photo yet.")
+
+    last_modified_timestamp = camera.last_photo_timestamp
+    last_modified = datetime.utcfromtimestamp(last_modified_timestamp)
+    last_modified_str = last_modified.strftime(HEADER_DATETIME_FORMAT)
+
+    # Give PrusaLink some time to take a new snapshot
+    timeout = PHOTO_TIMEOUT + TIME_FOR_SNAPSHOT
+
+    expires = str(last_modified + timedelta(seconds=timeout))
+
+    headers = {
+        'Last-Modified': last_modified_str,
+        'Date': last_modified_str,
```

Comment:
I think, that `Date` header should be datetime of response....
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date